src/openforms/emails/tests/test_confirmation_email.py

```diff
@@ -797,3 +797,128 @@ def test_templatetag_alias(self):
         ).lstrip()
 
         self.assertEquals(expected_text, text)
+
+    def test_email_with_default_design_token(self):
+        with patch(
+            "openforms.config.models.GlobalConfiguration.get_solo",
+            return_value=GlobalConfiguration(logo="https://logo.png"),
```

Comment:
Maybe good to make it explicit that here no design tokens are specified:
```suggestion
            return_value=GlobalConfiguration(logo="https://logo.png", design_token_values={}),
```